Dockerfile

```
@@ -0,0 +1,85 @@
+
+# --- stage 2: the image that ships ------------------------------------------------------------
+# Distroless: the binary plus its four shared libraries, with no shell and no package manager, so
+# the attack surface is the application rather than a distribution. `ldd` on the release binary
+# lists exactly libstdc++, libm, libgcc_s and libc, which is the whole reason this fits: nothing
+# else has to come along. 45 MB against 140 MB for the full-Debian form.
+#
+# **debian13, NOT debian12**, and this is load-bearing. The release is built on ubuntu-24.04
+# (glibc 2.39), so the binary requires glibc >= 2.38. The debian12/bookworm images ship 2.36, where
+# it installs cleanly and then dies at startup with "GLIBC_2.38 not found" from libc and libm.
+# Verified both ways on the bench. If the release ever moves to an older builder, this can too.
+FROM gcr.io/distroless/cc-debian13
```

Hadolint (2.15.1)

[warning] Always tag the version of an image explicitly

(DL3006)